src/Problem2Class.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+
+class Problem2Class
+{
+
+    public function test(): string
+    {
+        foreach ($_POST['menu-locations'] as $menu_id) {
```

Comment:
:warning: **Warning**: Detected usage of a non-sanitized input variable: $_POST['menu-locations'] (*WordPress.Security.ValidatedSanitizedInput.InputNotSanitized*).